include/aspect/particle/world.h

```diff
@@ -165,11 +178,13 @@ namespace aspect
          */
         void update_particles();
 
+#if !DEAL_II_VERSION_GTE(9,0,0)
         /**
```

Comment:
why are these inside an ifdef?

Reply:
Because all of the output functionality is not longer used in deal.II 9.0. We then use the `DataOut` class instead. I guess theoretically, I could leave out the ifdefs around the output functionality, and simply not use it anymore, but leaving them in reminds us that this is dead code and we can remove it once we require 9.0.